src/css/components/_loading.scss

```diff
@@ -20,8 +20,7 @@
   visibility: hidden;
 }
 
-.vjs-seeking .vjs-loading-spinner,
-.vjs-waiting .vjs-loading-spinner {
+.vjs-loading-spinner.show {
```

Comment:
the `.show` class is too generic. We generally prefix everything with `vjs`

Reply:
Changed in latest commit to .vjs-loading-spinner-show